client/sections-preloaders.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Internal dependencies
+ */
+import { preload } from './sections-helper';
+
+export function preloadEditor() {
+	preload( 'gutenberg-editor' );
+	preload( 'post-editor' );
```

Comment:
What if we preloaded only the `gutenberg-editor`? That's what a vast majority of users will really load today. The `post-editor` preload will be almost always wasted.

Reply:
I was hoping someone would suggest that! 🙌  Thanks!

Looking at the `selected-editor` stats, it seems like Calypso classic editor users are now down to 0.5% of all editor users, so it sounds like it's more than sensible to remove `post-editor` from being preloaded.

Addressed in https://github.com/Automattic/wp-calypso/pull/45934/commits/ed39e306e2ada200659bf4487bf504946882626f